results/proto/pipelinerun.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package tekton.pipeline.v1;
+
+import "common.proto";
+import "taskrun.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "./proto";
+
+message PipelineRun {
+    string api_version = 1;
+	string kind = 2; // TODO: must always be TaskRun?
```

Comment:
Can probably remove the TODO (or switch to PipelineRun to reflect the right type)